data/lib/core/storages.lua

```diff
@@ -43,4 +43,6 @@ PlayerStorageKeys = {
 
 	-- Bestiary:
 	bestiaryKillsBase = 400000,
+	bestiaryTrackerBase = 500000,
+	bestiaryTrackerMax = 502000,
```

Comment:
We need to reserve much more, there are already almost 2000 creatures in a Tibia datapack. Maybe bump up to 10k or reserve the entire 100k range (just like `bestiaryKillsBase` can now use)